docker/mypy.docker

```diff
@@ -6,15 +6,14 @@ RUN docker-apt-install \
     gir1.2-notify-0.7 \
     libnotify4 \
     python3-gi \
-    python3-nacl \
+    python3-dbus \
```

Comment:
nacl is a requirement right? why remove this.

Reply:
That's not intentional, thanks for your remark.